build.gradle

```diff
@@ -262,7 +263,7 @@ pitest {
   historyOutputLocation = 'build/reports/pitest/fastermutationtestingoutput'
   outputFormats = ['XML', 'HTML']
   timestampedReports = true
-  mutationThreshold = 20
+  mutationThreshold = 70
 }
```

Comment:
not a comment for this pr but reminder to bring the threshold upto 90%

Reply:
yes we have separate ticket for this